consensus/types/src/attestation.rs

```diff
@@ -104,6 +89,34 @@ impl<E: EthSpec> Hash for Attestation<E> {
 }
 
 impl<E: EthSpec> Attestation<E> {
+    pub fn empty_for_signing(
+        committee_index: usize,
+        committee_length: usize,
+        data: AttestationData,
+        spec: &ChainSpec,
+    ) -> Result<Self, Error> {
+        if spec.fork_name_at_slot::<E>(data.slot) >= ForkName::Electra {
+            let mut committee_bits: BitVector<E::MaxCommitteesPerSlot> = BitVector::default();
+            committee_bits
+                .set(committee_index, true)
+                .map_err(|_| Error::InvalidCommitteeIndex)?;
+            Ok(Attestation::Electra(AttestationElectra {
+                aggregation_bits: BitList::with_capacity(committee_length)
+                    .map_err(|_| Error::InvalidCommitteeLength)?,
```

Comment:
`empty_for_signing` is used in the validator client where the values from duties are not trusted. Current code panics in that case. This PR switches to return errors.